Doc 851 (#991)
amplitude/amplitude-docs: content/collections/data/en/autocapture.md

* Autocapture’s default settings further restrict your collection of sensitive input fields, like passwords or form fields with the hidden attribute, and only captures class and type attribute values. Autocapture doesn't capture other details about these elements, including any of the content of the input fields an end user may populate.
* Autocapture captures the text your website or app displays. For example, the content (`textContent`) of the element clicked and its children. It's not recommended that you use Autocapture's element interaction tracking on pages that may contain sensitive information. Amplitude uses pattern matching to automatically exclude from your collection any text content that looks like a credit card number, social security number, or email address.
* The exception to these attribute collection rules is when an element has an explicit attribute added with the prefix *data-amp-track-*. This allows data in these attributes to be intentionally passed back to Amplitude.
* Autocapture automatically removes value, event handlers, style, and react attributes.

> Additional autocapture protections include: 

Reviewer: Lets maybe add a topic to discuss using headings as list items. I don't have an issue with it, but so far we use H2-H4 exclusively to communicate hierarchy

#### Precise text and title masking

> Precise text, otherwise known as title masking, excludes text capture from specific elements on the page, including the page title. This lets Autocapture usage on the page without capturing the text displayed in the specific element.

Reviewer: ```suggestion
Precise text, otherwise known as title masking, excludes text capture from specific elements on the page, including the page title. This lets you use Autocapture on the page without capturing the text displayed in specific elements.
```

> To prevent capturing the text, add the attribute `data-amp-mask` to the element. If, for example you have the following on a button: 

Reviewer: ```suggestion
To prevent capturing an element's text, add the attribute `data-amp-mask` to the it. If, for example you have the following on a button: 
```
Small thing, but an example using a text input or a div (as is shown) might work better.

```html
<div data-amp-mask>John Doe</div>
```
The above example still tracks the click on the div. However, the text content "John Doe" doesn't appear in the Autocapture data. This exclusion works on the SDK level. 

Precise text works recursively so that any text contained in an element is masked. For example: 

```html
<div data-amp-mask>
  John Doe
  <div>Jane Doe</div>
</div>
```
In the above example, both names are masked with `*****`.

#### Email address masking

Email address masking automatically hides or otherwise obscures email addresses. Amplitude uses pattern matching to automatically exclude content that looks like an email address. Email address masking is recursive. Any email addresses captured within an element is masked such as `*****`.

#### Pattern (Regex) masking

You can mask information based on patterns that you specify (regular expressions or Regex). If you aren't familiar with Regex, review this page on [regular expressions](https://www.regular-expressions.info/quickstart.html).

> You can specify a pattern of information that you want to mask within Amplitude. This configuration occurs in the SDK and is an additional layer of protection to the default patterns Amplitude uses to mask email, credit cards, and social security numbers. Regex filters mask values in any fields where it may be possible to include this data. This includes both visible fields as well as hidden attributes on the page. 

Reviewer: ```suggestion
Specify a pattern of information that you want Amplitude to mask. This configuration happens in the SDK and is an additional layer of protection to the default patterns Amplitude uses to mask email, credit cards, and social security numbers. Regex filters mask values in any fields where it may be possible to include this data. This includes both visible fields as well as hidden attributes on the page. 
```
A link to the SDK doc sections wherever you reference the SDK will be helpful.

> For example, you can set a Regex pattern to filter account numbers such as "#0236732." If Amplitude finds account numbers that match the pattern you set, those numbers are masked such as: `*****`. 

Reviewer: Can you include an example regex?

#### Page URL allow list and block list

Page URL block- and allow- lists let you specify unique URL page patterns to either be excluded or included from Amplitude. 

For example, you can exclude user activity from specific sub-domains on your URL such as from your user's account settings or URLs that only include testing data. 

> Alternately, if host your site in multiple domains such as .com and .co.uk, you could specify that you only want to collect data from the .co.uk domain. In this scenario, you'd add the .co.uk domain to the allow list.

Reviewer: ```suggestion
Alternately, if you host your site in multiple domains such as .com and .co.uk, you could specify that you only want to collect data from the .co.uk domain. In this scenario, you'd add the .co.uk domain to the allow list.
```

#### Precise attribute redaction

> The precise attribute redaction excludes specific elements from capture. This lets you use Autocapture on a page without capturing attributes in a specified element. 

Reviewer: ```suggestion
Precise attribute redaction excludes specific elements from capture. This lets you use Autocapture on a page without capturing attributes in a specified element. 
```
Removed "The" as none of the other sections lead with that.
